conf/nutch-default.xml

```diff
@@ -1136,6 +1136,28 @@
 <!-- parser properties -->
 
 <property>
+  <name>parser.modelfilter.trainfile</name>
+  <value>tweets-train.tsv</value>
+  <description>
```

Comment:
I will update the code for all the formatting and info changes. I had already mentioned it in jira. This is an "Uncouth" patch just to show the flow and the proof of concept. Thanks for the formatting pointers though :)